sandbox-templates/svelte-developer/e2b.Dockerfile

```
@@ -0,0 +1,25 @@
+# Use Node.js base image for Svelte
+FROM node:21-slim
+
+# Install necessary system dependencies
+RUN apt-get update && apt-get install -y curl && apt-get clean && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)